src/geckomat/gather_kcats/getStandardKcat.m

```diff
@@ -126,69 +133,88 @@
 % Find reactions without GPR
 rxnsMissingGPR = find(cellfun(@isempty, model.grRules));
 
+% Get custom list of reaction IDs to ignore, if existing. First column
+% contains reaction IDs, second column contains reaction names for
+% reference only.
+if exist(fullfile(params.path,'data','pseudoRxns.tsv'),'file')
+    fID        = fopen(fullfile(params.path,'data','pseudoRxns.tsv'));
+    fileData   = textscan(fID,'%s %s','delimiter','\t');
+    fclose(fID);
+    customRxns = fileData{1};
+    customRxns = find(strcmp(customRxns,model.rxns));
```

Comment:
Error using `strcmp`
Inputs must be the same size or either one can be a scalar.

Reply:
```suggestion
    customRxns = find(ismember(model.rxns,customRxns));
```